mlir/lib/IR/AffineExpr.cpp

```diff
@@ -237,6 +237,149 @@ unsigned AffineDimExpr::getPosition() const {
   return static_cast<ImplType *>(expr)->position;
 }
 
+/// Returns true if the expression is divisible by the given symbol with
+/// position symbolPos.
```

Comment:
Use backticks around `symbolPos`.

Reply:
`exprKind` is not documented nor is it clear what its role is from the signature.